hackathon_site/registration/models.py

```diff
@@ -99,9 +105,30 @@ class Application(models.Model):
         ],
         null=False,
     )
-    q1 = models.TextField(null=False, help_text="First question?", max_length=1000)
-    q2 = models.TextField(null=False, help_text="Second question?", max_length=1000)
-    q3 = models.TextField(null=False, help_text="Third question?", max_length=1000)
+    linkedin = models.CharField(
```

Comment:
you can use django's URLField here instead for all the links
https://docs.djangoproject.com/en/4.1/ref/models/fields/#urlfield